File: app/src/main/java/io/github/zwieback/familyfinance/core/drawer/DrawerListener.java
package io.github.zwieback.familyfinance.core.drawer;

import android.app.Activity;
import android.view.View;
import androidx.annotation.NonNull;

import com.mikepenz.materialdrawer.Drawer;
import com.mikepenz.materialize.util.KeyboardUtil;

public class DrawerListener implements Drawer.OnDrawerListener {

    private final Activity activity;

    public DrawerListener(Activity activity) {
        this.activity = activity;
    }

    @Override
    public void onDrawerOpened(@NonNull View drawerView) {
        KeyboardUtil.hideKeyboard(activity);
    }

    @Override
    public void onDrawerClosed(@NonNull View drawerView) {
        // stub
    }

    @Override
    public void onDrawerSlide(@NonNull View drawerView, float slideOffset) {
        // stub
    }
}

File: app/src/main/java/io/github/zwieback/familyfinance/core/drawer/DrawerListener.kt
package io.github.zwieback.familyfinance.core.drawer

import android.app.Activity
import android.view.View

import com.mikepenz.materialdrawer.Drawer
import com.mikepenz.materialize.util.KeyboardUtil

class DrawerListener(private val activity: Activity) : Drawer.OnDrawerListener {

    override fun onDrawerOpened(drawerView: View) {
        KeyboardUtil.hideKeyboard(activity)
    }

    override fun onDrawerClosed(drawerView: View) {
        // stub
    }

    override fun onDrawerSlide(drawerView: View, slideOffset: Float) {
        // stub
    }
}
